addons/event/models/event_mail.py

```diff
@@ -114,13 +119,48 @@ def execute(self):
                 self.write({'mail_sent': True})
         return True
 
+    @api.model
+    def _warn_template_error(self, scheduler, exception):
+        if random.random() < 0.15:
```

Comment:
Isn't there a chance that nobody will ever be noticed about the issue? Is it really a problem to notify them every time?
And if this frequency is sufficient, it would be nice to add a small comment explaining why ;-)